meta/WriteOut.m

```diff
@@ -1227,6 +1227,10 @@
 {
    for (const auto& particle : decay_table) {
       set_decay_block(particle, flexibledecay_settings);
+      const int id = particle.get_particle_id();
+      if ((id % 25) % 10 == 0 || (id % 36) % 10 == 0 || (id % 37) % 10 == 0) {
```

Comment:
I was just trying to wrap my head around this condition and could not fit it to the PDG numbers I know (in particular due to the `%10` operation. Would you mind adding a comment or giving the condition a name like (don't know if this is the correct name)
~~~.cpp
const bool is_higgs_boson = (id % 25) % 10 == 0 || (id % 36) % 10 == 0 || (id % 37) % 10 == 0;
~~~

Reply:
This was a quick test. I think the proper syntax is `(id - 25) % 10` etc :smiling_face_with_tear: